highfidelity/hifi
# Remove mouse area for text fields. use activeFocusOnPress property in…

…stead

Based on: https://highfidelity.fogbugz.com/f/cases/7973/Keyboard-focus-problems-within-Tablet-HUD

Test 
- Enable Commerce as described in the case
- Try to setup Wallet in Desktop mode
- on step 3 of 4 with "Got it" button 1st press outside wallet setup window (loose focus)
- Press "Got it"
- Press inside password input fileds
- Make sure input fields gained focus
